### PR TITLE
Add Eslint, Prettier, Husky

### PR DESCRIPTION
**Pull request by vj-abishek**
This PR solves #19 this problem.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,23 @@
   },
   "scripts": {
     "test": "jest",
-    "start:dev": "nodemon src/app.js"
+    "start:dev": "nodemon src/app.js",
+    "prettier": "prettier \"**/*.{js,jsx,css,json}\"",
```
Comment (YashKumarVerma) [inline package.json:12]: we won't be having any .css files as this is an api

File: package.json [hunk 2]
```diff
@@ -8,7 +8,23 @@
...
+    "lint:fix": "npm run lint -- --fix",
+    "validate": "npm run prettier -- --list-different && npm run lint"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "**/*.{js,jsx,css}": [
```
Comment (YashKumarVerma) [inline package.json:24]: css, jsx not required.

**Review by YashKumarVerma — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (vj-abishek) [reply package.json:24]: Okay, I will fix it.

**Comment by vj-abishek** [comment]
@YashKumarVerma There is an issue with package-lock.json. So It is getting difficult to resolve the conflict. 

Annotation 2020-10-06 213442

I tried in my local machine and  `gitpod.io`. In both platforms, it is not working.
I cannot even install the package. Only getting this issue from recent PR

**Comment by YashKumarVerma** [comment]
Requesting to resolve merge conflicts.

**Comment by vj-abishek** [comment]
Working on it

**Comment by vj-abishek** [comment]
@YashKumarVerma There is an error in the database connection. So disabling `eslint` and `test` in pre-commit hook.

**Review by YashKumarVerma — APPROVED** [review]

**Comment by lopeselio** [comment]
Changes have been merged

**Pull request merged**